<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>readFromCameraWindow</class>
 <widget class="QMainWindow" name="readFromCameraWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>771</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>387</width>
    <height>771</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyQRCode Reader(Camera)</string>
  </property>
  <property name="windowIcon">
   <iconset theme=":/appLogo/icon.png"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(16, 100, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleReaderCamera">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>PyQRCode Reader(Camera)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="videoSource">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="sourcelabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Source:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readCamera">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 0, 94);
color: rgb(251, 226, 255);</string>
    </property>
    <property name="text">
     <string>Read Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Data:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="dataRead">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>351</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 253, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="opText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>640</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>If you want to save the data as a text click on this button.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveasText">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>650</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 0, 94);
color: rgb(251, 226, 255);</string>
    </property>
    <property name="text">
     <string>Save as Text</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>387</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
